js/RateCurve.js

```diff
@@ -1,8 +1,12 @@
 'use strict';
 
-var minRc = 1000;
-var midRc = 1500;
-var maxRc = 2000;
+var rcParams = () => {
```

Comment:
For easier readability, I'd prefer if we stuck to 'classical JS' function definitions `var rcParams = function () {...}`. Or else switched all of them to hashrocket style.